dependencies/README.md

```diff
@@ -0,0 +1,62 @@
+Static Linkage Checker
+======================
+
+Static Linkage Checker is a tool that inspects classpaths for [static linkage check](
+../library-best-practices/glossary.md#static-linkage-check).
+The tool identifies static linkage errors that appear in Java projects at runtime.
+It scans the class files in a classpath provided as input for references
+to other classes. Then it verifies compatibility of the methods and fields
+in the referenced classes, with regard to their type signatures and accessors.
+
+### Use Cases
+
+There are three use cases for Static Linkage Checker:
+
+-  **For organizations** that provide multiple libraries developed by different teams,
+  the tool helps to ensure that there are no static linkage conflicts among the libraries and their
+  dependencies when some of them are used at the same time.
+
+- **For library developers**, the tool ensures that the users of the library will not suffer
+  static linkage conflicts caused solely by the dependencies of the library.
+
+- (future plan) **For application developers** consuming libraries, the tool will
```

Comment:
> I placed "future plan" to indicate this use case is out of scope of 2018 Q4 target.

Honestly, completion of any part of the tool is out of scope of 2018 Q4 target.

As for usage by library developers and application developers, I feel like there is really no difference. We either support both or neither.

I feel like "future plan" makes use for calling out future functionality; it doesn't seem like tool documentation is a good place talk about future support for different classes of users.